Week-1/assignment-2/index.html

```diff
@@ -0,0 +1,108 @@
+
+                            <div class="songs-options">
+                                <span>
+                                    <i class="las la-play"></i> Play all
+                                </span>
+                                <span>
+                                    <i class="las la-plus"></i> Add to
+                                </span>
+                                <span>
+                                    <i class="las la-ellipsis-h"></i> More
+                                </span>
+                            </div>
+                        </div>
+                    </div>
+
+
+                    <table class="album-table">
+                        <tbody>
+                            <tr>
+                                <td>1. One Dance</td>
+                                <td>Crate feat Catlid & Cyla</td>
+                                <td>2:54</td>
+                                <td> <span><a href="http://google.com"> CATTY CLOUD SYNC</a> </span> </td>
+                            </tr>
+                            <tr>
+                                <td>2. Panda</td>
+                                <td>Cattee</td>
+                                <td>4:06</td>
+                                <td><span><a href="http://google.com"> CATTY CLOUD SYNC</a> </span></td>
+                            </tr>
+                            <tr>
+                                <td>3. Can't stop feeling! comercial Moctezuma</td>
+                                <td>Francisco Chang</td>
+                                <td>3:56</td>
+                                <td><span> <a href="http://google.com">CATTY CLOUD SYNC</a> </span></td>
+                            </tr>
+                            <tr>
+                                <td>4. Work from home</td>
+                                <td>Cat Harmony feat colla</td>
+                                <td>3:34</td>
+                                <td><span> <a href="http://google.com">CATTY CLOUD SYNC</a> </span></td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </section>
+
+            <div class="footer-playbox">
+                <div class="song-favicon">
+                    <img src="https://vif1g.csb.app/src/assets/cattyboard.jpg" alt="song-favicon">
+                    <div class="more-info">
+                        <p>If it ain't love</p>
+                        <p>Catson Derulo</p>
+                    </div>
+                </div>
+
+                <div class="song-controlls">
+                    <i class="las la-step-backward"></i>
+                    <i class="las la-pause"></i>
+                    <i class="las la-step-forward"></i>
+                    <i class="las la-undo-alt"></i>
+                    <i class="las la-random"></i>
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
+
+</html>
```

Comment:
@chinmay4o Please use new lines at end of each file. Configure your formatter accordingly. Here is the explanation for it https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b